src/components/HeroHeader/index.jsx

```diff
@@ -1,11 +1,28 @@
 import React from 'react'
 import { useClient } from 'cozy-client'
 import { useInstanceSettings } from 'hooks/useInstanceSettings'
+import cx from 'classnames'
 
+import { useCozyTheme } from 'cozy-ui/transpiled/react/CozyTheme'
+import { makeStyles } from 'cozy-ui/transpiled/react/styles'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 
+const useStyles = makeStyles(theme => ({
+  shadow: {
+    boxShadow: theme.shadows[2]
+  },
+  nameInverted: {
+    textShadow: theme.textShadows[1]
+  },
+  hostInverted: {
+    textShadow: theme.textShadows[1]
+  }
+}))
```

Comment:
There is no "public API" for that? I find it hard to believe that an application has to know about `shadows[2]` & `textShadows[1]`. No component / no variants already exist?